docs/reference/pf-command-reference.md

```diff
@@ -4,6 +4,14 @@
 This is an experimental feature, and may change at any time. Learn [more](../how-to-guides/faq.md#stable-vs-experimental).
 :::
 
+## Autocomplete
+
+To activate autocomplete features for the pf CLI you need to add the following snippet to your ~/.bashrc or ~/.zshrc:
```

Comment:
direct copy this into rc file?

Reply:
You mean "echo and append" it to the rc ? The problem is that we need to add logic that detects where your rc is and how it is called. In an XDG desktop it will be under ~/.config and in non XDG it will be under ~

Another issue is that if we have different "venvs" and have "pf" installed multiple times, which autocomplete should be used ?